internal/server/data/migrations.go

```diff
@@ -637,3 +619,19 @@ func deleteDuplicates(tx *gorm.DB, group string, model models.Modelable) error {
 	subQuery := tx.Select("min(id)").Group(group).Model(model)
 	return tx.Where("id NOT in (?)", subQuery).Delete(model).Error
 }
+
+// nolint:revive
+func addFieldsFor_0_14() *migrator.Migration {
```

Comment:
Now that we don't call `AutoMigrate` after `migrate` we will need to explicit create migrations for adding columns.

I think this is much safer. It'll prompt us to write tests for them as well, and it will be more clear that we are making changes to the db schema.

The downside is that it's currently a lot of work to add a test for a database migration, so I'm going to be looking at making that easier next.